docs/during/timetool.md

```diff
@@ -18,7 +19,15 @@ Before the time tool can be employed it must be calibrated. During the process o
 
 (TBA Thursday 2023/03/02)
 
-## Analysis
+![Timetool - DAQ control interface](./material/DAQ_control_TT.png)
+*DAQ control interface to the Timetool. In the `DAQ Control` panel, (1) click `Edit`, (2) click `OPAL1K_1` or some equivalent device name, (3) click on `TimeToolConfig`. This will open the window on the left.*
+
+![Timetool - DAQ partition interface](./material/DAQ_select_TT.png)
+*DAQ control interface to the partition. To select what is recorded in the DAQ partition, (1) click `Select`, (2) click `opal_tt` to record the timetool raw image data.*
+
+<div align="right">&#8673; <a href="#" >Back to Top</a></div>
```

Comment:
Same thing for using the markdown tags. The headers titles should also work as anchors without extra tags, so we could probably link to the top header?

Reply:
Good point. Done. (we still need the html align though...).